README.md

```diff
@@ -141,6 +141,10 @@ Ramda versions > 0.25 don't have a default export.
 So instead of `import R from 'ramda';`, one has to use `import * as R from 'ramda';`
 Or better yet, import only the required functions via `import { functionName } from 'ramda';`
 
+**Note for ES6 module and browsers**
+In order to access to the ES6 module in browsers, one has to provide the content of  the __es__ directory (see below for the build instructions) and use `import * as R from '<path-to-es-directory>/es/index.js';`
+
```

Comment:
Should that be `path-to-ramda-directory` ?

Reply:
I'm referring to the es directory created in the build phase

Reply:
This simply doesn't make sense to me:

```
import * as R from '<path-to-es-directory>/es/index.js';`
//                   ^^^^^^^^^^^^^^^^^^^^
```

What es directory are we talking about here?  The `es` directory is inside the Ramda one, right?

```
import * as R from '<path-to-es-directory>/es/index.js';`
//                                         ^^
```

Reply:
When using `npm install ramda` and otherwise working build-tool free in the browser, it's exactly as you suspect @CrossEye 

```javascript
import * as R from './node_modules/ramda/es/index.js';
```

Reply:
ok done